runner/src/internal.rs

```diff
@@ -62,6 +49,10 @@ pub struct RunExamples {
         default_value = "rust"
     )]
     pub application_variant: String,
+    #[structopt(long, help = "name of a single example to run")]
+    pub example_name: Option<String>,
```

Comment:
Interesting, so the way to run one example is to run `RunExamples`, but with an option to run actually just one. 
Similarly to build an example ones one runs `RunExamples` but uses a bool to indicate to actually just build. 

It feels a bit counterintuitive, but I assume it's done for less code duplication. 

One thing that would seem more intuitive to me would be use a `example_names` Vec, as opposed to a single string. To run a single example, one would then pass a Vec with just that entry.

Reply:
Makes sense :). My intuition is always to make API as friendly to future iteration with things like that, but admittedly a lot of that comes from working in a dynamic lang, which makes changing them down the road much harder.